src/PageContainer.js

```diff
@@ -65,12 +80,25 @@ class PageContainer extends Component {
                 {this.state.content === PageContent.CALENDAR && (
                     <div style={{ marginTop: '120px', marginLeft: '250px' }}>
                         Calendar
-                        <EventCard
-                            eventType={DeliveryType.RECURRING}
-                            startTime="10am"
-                            endTime="12pm"
-                            futureEvent={true}
-                        />
+                        {this.state.dialogOpen ? (
+                            <Dialog
+                                closeDialog={this.closeDialog}
+                                eventType={DeliveryType.RECURRING}
+                                accountType={this.props.account.accountType}
+                                date="11/14/2017"
+                                startTime="10am"
+                                endTime="12pm"
+                                futureEvent={false}
```

Comment:
actually, on a second thought, I think you should mock dummy data in `componentDidMount()`:
```
// TODO move this dummy data mocking to calendar
let delivery = {
    eventType: DeliveryType.RECURRING,
    date: "11/14/2017",
    startTime="10am",
    endTime="12pm",
}
```
and only pass in
```
<Dialog
    closeDialog={this.closeDialog}
    accountType={this.props.account.accountType}
    delivery={this.state.delivery}
    futureEvent={false}
/>
```
here.